libs/core/langchain_core/prompts/chat.py

```diff
@@ -901,19 +929,31 @@ def from_messages(
             partial_variables=partial_vars,
         )
 
-    def format(self, **kwargs: Any) -> str:
```

Comment:
Removed `format` from `ChatPromptTemplate` as it is identical to the one in the parent class `BaseChatPromptTemplate `